content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -162,12 +211,46 @@ The following manifest defines an Ingress that sends traffic to your Service via
    ```
 
 
-1. Verify that the Ingress controller is directing traffic:
+1. Verify that the Ingress controller is directing traffic, by following the instructions for your platform:
+
+   {{< note >}}
+   The network is limited if using the Docker driver on MacOS (Darwin) and the Node IP is not reachable directly. To get ingress to work you’ll need to open a new terminal and run `minikube tunnel`.  
```

Comment:
It should be the same for macOS and Windows, right?

Reply:
Yes.
Initially the PR catered to both MacOS and Windows. I was looking for a collaborator to help me test the Windows related changes, but wasn't able to find a volunteer.

As mentioned in the comments above, the scope of the PR has been reduced to cater to just the MacOS part, as that is on the critical path (fixes 2 open issues) and I have successfully tested the PR instructions for MacOS.